AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -162,6 +162,11 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
  */
 @property(nonatomic) MSDistributeFlags distributeFlags;
 
+/**
+ * Flag to indicate whether an update was started with a checkForUpdate call.
+ */
+@property(atomic) bool checkForUpdateFlag;
```

Comment:
We can get rid of this property and add a parameter to `startUpdate` for a flag that inforce update flow even though automatic check is disabled. We don't care authentication stuff at the moment so we can simplify this one.